DW CLI doesn't resolve folder paths as expected in certain terminals/shells (#43)
dynamicweb/CLI: README.md

#### Examples
Changing the host for the dev environment
> $ dw config --env.dev.host localhost:6001

## Using Git Bash
If you're using Git Bash, you may encounter issues with path conversion that can interfere with relative paths used in commands.

### Path Conversion Issues
Git Bash automatically converts relative paths to absolute paths, which can cause problems. 
You'll see a warning message if the conversion setting is not disabled:

> "You appear to have path conversion turned on in your shell. If you are using relative paths, this may interfere. Please see doc.dynamicweb.dev for more information."

Reviewer: ```suggestion
"You appear to have path conversion turned on in your shell. If you are using relative paths, this may interfere. Please see https://doc.dynamicweb.dev/[link-to-cli-docs] for more information."
```

Author: Done

### Solution
To resolve this issue, disable path conversion by setting the `MSYS_NO_PATHCONV` environment variable (current session only):

> $ export MSYS_NO_PATHCONV=1

#### Examples

> $ export MSYS_NO_PATHCONV=1
> $ dw files -iro ./ /TestFolder --host \<host\> --apiKey \<apiKey\>

### Alternative Solutions
If you prefer not to disable path conversion globally, you can:

> 1. Prefix problematic paths with `//` to prevent conversion for specific commands.

Reviewer: Don't suggest double slash. It's not usually used in paths as slashes have special meaning. For clarity, use `./` instead.
```suggestion
1. Prefix relative paths with `./` instead of just `/` to prevent conversion for specific commands.
```

Author: Done

2. Use PowerShell or CMD instead of Git Bash.

#### Examples

> > $ dw files -iro ./ //TestFolder --host \<host\> --apiKey \<apiKey\>

Reviewer: ```suggestion
> $ dw files -iro ./ ./TestFolder --host \<host\> --apiKey \<apiKey\>
```

Author: Done
